ext/reflection/tests/ReflectionExtension_getClasses_basic.phpt

```diff
@@ -7,120 +7,125 @@ Thijs Lensselink <tl@lenss.nl>
 $ext = new ReflectionExtension('reflection');
 var_dump($ext->getClasses());
 ?>
---EXPECT--
-array(23) {
+--EXPECTF--
```

Comment:
Most of this change was just to make future changes a bit easier, it can be committed separately, if necessary ?